src/library.js

```diff
@@ -1493,26 +1493,6 @@ LibraryManager.library = {
     return 0;
   },
 
-  // ==========================================================================
-  // sys/types.h
-  // ==========================================================================
-  // http://www.kernel.org/doc/man-pages/online/pages/man3/minor.3.html
-  makedev__sig: 'iii',
-  makedev: function(maj, min) {
-    return ((maj) << 8 | (min));
-  },
-  gnu_dev_makedev: 'makedev',
-  major__sig: 'ii',
-  major: function(dev) {
-    return ((dev) >> 8);
-  },
-  gnu_dev_major: 'major',
-  minor__sig: 'ii',
-  minor: function(dev) {
-    return ((dev) & 0xff);
-  },
-  gnu_dev_minor: 'minor',
```

Comment:
How about keeping the `gnu` aliases? I don't see those in the header. I imagine they were added here because some codebase needed them.

But, I suppose that would require deps_info work, which may not be worth it, so lgtm.

Reply:
These function don't exist on the native side either, they are just macros.

The gnu_dev_minor, gnu_dev_major and gnu_dev_makedev symbols are glibc-only functions.  I can't see any reason to support them since they don't appear in any headers.   I'll do a bit more digging, but we certainly don't have any tests for these as of today.

Reply:
These were first introduced back in d1ed9565a7e3dc28e7012a2bdfa794d8aa88ad49,  it looks like for compatibility if CPython.

However, looking at the cpython source code I don't see any reference to any of these three functions either on tip of tree of historically.    The only way I can imagine these getting into a project would be if somebody somehow used glibc headers instead of musl..

Reply:
I vote we simple remove this stuff, since its untested and AFAICT unused anywhere.